jupyter/analysis-automation/BasicExample.ipynb

```diff
@@ -0,0 +1,456 @@
+      "text/plain": [
+       "<Figure size 1080x720 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Create plot and print data \n",
+    "fig, (ax1, ax2) = plt.subplots(nrows=2, ncols=1, figsize=(15, 10))\n",
+    "fig.suptitle ('Temperature Monitoring File: '+ test_file_name + '     Test program: ' + program_name, fontsize=curr_fontsize, color='blue')\n",
+    "\n",
+    "ax1.set_title(group_names[1], fontsize=curr_fontsize)\n",
+    "plot_area(ax1, bulk_data[1],  channels[1], False, formatted_properties[1])\n",
+    "ax2.set_title(group_names[0], fontsize=curr_fontsize)\n",
+    "plot_area(ax2, bulk_data[0], channels[0], True, formatted_properties[0])\n",
+    "\n",
+    "plt.tight_layout()\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Add Result Summary\n",
+    "\n",
+    "Each Scrap recorded with `sb.glue()` is displayed for each procedure on the __History__ tab in Analysis Automation. "
```

Comment:
Scrap -> scrapbook?

Reply:
According to the documentation of scrapbook, they call it "Scrap":

https://nteract-scrapbook.readthedocs.io/en/latest/usage-glue.html#glue-api
https://nteract-scrapbook.readthedocs.io/en/latest/models.html#scrap